### ISSUE TITLE
failed with: OpenAPI spec does not exist

### ISSUE THREAD

**Issue by moechs** [imroc/cert-manager-webhook-dnspod#17]
### Versions

* Kubernetes: v1.26.10+k3s2
* Cert-manager: v1.13.2

### Logs

```bash
controller.go:129] OpenAPI AggregationController: action for item v1alpha1.acme.imroc.cc: Rate Limited Requeue.
controller.go:116] loading OpenAPI spec for "v1alpha1.acme.imroc.cc" failed with: OpenAPI spec does not exist
controller.go:129] OpenAPI AggregationController: action for item v1alpha1.acme.imroc.cc: Rate Limited Requeue.
controller.go:116] loading OpenAPI spec for "v1alpha1.acme.imroc.cc" failed with: OpenAPI spec does not exist
controller.go:129] OpenAPI AggregationController: action for item v1alpha1.acme.imroc.cc: Rate Limited Requeue.
controller.go:116] loading OpenAPI spec for "v1alpha1.acme.imroc.cc" failed with: OpenAPI spec does not exist
controller.go:129] OpenAPI AggregationController: action for item v1alpha1.acme.imroc.cc: Rate Limited Requeue.
controller.go:116] loading OpenAPI spec for "v1alpha1.acme.imroc.cc" failed with: OpenAPI spec does not exist
controller.go:129] OpenAPI AggregationController: action for item v1alpha1.acme.imroc.cc: Rate Limited Requeue.
controller.go:116] loading OpenAPI spec for "v1alpha1.acme.imroc.cc" failed with: OpenAPI spec does not exist
controller.go:129] OpenAPI AggregationController: action for item v1alpha1.acme.imroc.cc: Rate Limited Requeue.
controller.go:116] loading OpenAPI spec for "v1alpha1.acme.imroc.cc" failed with: OpenAPI spec does not exist
controller.go:129] OpenAPI AggregationController: action for item v1alpha1.acme.imroc.cc: Rate Limited Requeue.
```

Other community discussions about this issue：

* https://github.com/cert-manager/webhook-example/issues/27
* https://github.com/ns1/cert-manager-webhook-ns1/issues/23

Thank you for all your work on this repository.  and the blog https://imroc.cc/ very nice ^^

**Comment by imroc** [comment]
It sames related to cert-manager and webhook example template version, will upgrade to latest and debug when I have time.

**Comment by imroc** [comment]
v1.3.0 is released, you can update image tag to v1.3.0.

**Comment by moechs** [comment]
Thanks for the quick response! I've already tested version v1.3.0, and it seems like the issue has been resolved.
